src/request.ts

```diff
@@ -149,6 +149,9 @@ export async function download(url: string, destPath: string, overrides?: AxiosR
     url,
     method: 'GET',
     responseType: 'stream',
+    headers: {
+      Accept: 'application/octet-stream',
+    },
```

Comment:
As specified in the spec that's the header we should pass. I didn't check whether `responseType` sets any header, but without the later it wouldn't work on SQ